mubench.pipeline/tests/tasks/implementations/test_publish_findings_task.py

```diff
@@ -273,6 +289,24 @@ def test_converts_directed_graphs(self, replace_mock, post_mock):
 
         assert_equals([self.test_detector_execution.findings_path + "/graph"], post_mock.call_args[1]["file_paths"])
 
+    @patch("tasks.implementations.publish_findings.PublishFindingsTask._convert_graphs_to_files")
+    def test_publish_successful_run_in_sized_chunks(self, convert_mock, post_mock, getsize_mock):
+        self.uut.max_files_per_post = 100
+        self.uut.max_file_size_per_post = 1500
+        getsize_mock.return_value = 1024
+        self.test_detector_execution.is_success = lambda: True
+        self.test_potential_hits = PotentialHits([
+            self._create_finding({"rank": "-1-"}, convert_mock, file_paths=["-file1-"]),
+            self._create_finding({"rank": "-2-"}, convert_mock, file_paths=["-file2-"])
+        ])
+
+        self.uut.run(self.project, self.version, self.test_detector_execution, self.test_potential_hits,
+                     self.version_compile, self.detector)
+
+        assert_equals(len(post_mock.call_args_list), 2)
+        assert_equals(["-file1-"], post_mock.call_args_list[0][1]["file_paths"])
+        assert_equals(["-file2-"], post_mock.call_args_list[1][1]["file_paths"])
```

Comment:
Do we even need files in this test? Its name says it's supposed to only test the chunking by size, so I would remove the files from the findings and the last two assertions.